resources/upgrading-to-aep/current-sdk-versions.md

```diff
@@ -112,12 +112,36 @@ Adobe Experience Platform Mobile SDK for Android supports Apple iOS 10 or later
 
-## React Native
+## React Native (AEP 1.x) <a id="react-native-aep"></a>
+
+Adobe Experience Platform Mobile SDK plugin for React Native supports React Native **version 0.60.0 or later**. For the latest installation instructions, see the `README` file in the [`aepsdk-react-native`](https://github.com/adobe/aepsdk-react-native#installation) repository.
+
+{% hint style="danger" %}
+Adobe Experience Platform Mobile SDK plugins (AEP 1.x) for React Native are compatible only with [Android](current-sdk-versions.md#android) and [iOS — Swift](current-sdk-versions#ios-swift) libraries.
+{% endhint %}
+
+{% hint style="info" %}
+If you are currently using our ACP-prefix React Native libraries, please see [the guide to migrate to the Experience Platform SDK libraries (AEP 1.x)](https://github.com/adobe/aepsdk-react-native/blob/main/docs/migration.md) for React Native.
+{% endhint %}
+
+| Extension | npmjs |
+| ---- | ---- |
+| [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core) |[![npm version](https://img.shields.io/npm/v/@adobe/react-native-aepcore.svg?color=green&label=%40adobe%2Freact-native-aepcore&logo=npm&style=flat-square)](https://badge.fury.io/js/%40adobe%2Freact-native-aepcore)   |
+|  [Profile](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/profile)   |   [![npm version](https://img.shields.io/npm/v/@adobe/react-native-aepuserprofile.svg?color=green&label=%40adobe%2Freact-native-aepuserprofile&logo=npm&style=flat-square)](https://badge.fury.io/js/%40adobe%2Freact-native-aepuserprofile) |
+|  [Edge](https://aep-sdks.gitbook.io/docs/foundation-extensions/experience-platform-extension)    |   [![npm version](https://img.shields.io/npm/v/@adobe/react-native-aepedge.svg?color=green&label=%40adobe%2Freact-native-aepedge&logo=npm&style=flat-square)](https://badge.fury.io/js/%40adobe%2Freact-native-aepedge)   |
+|  [EdgeIdentity](https://aep-sdks.gitbook.io/docs/foundation-extensions/identity-for-edge-network)    |     [![npm version](https://img.shields.io/npm/v/@adobe/react-native-aepedgeidentity.svg?color=green&label=%40adobe%2Freact-native-aepedgeidentity&logo=npm&style=flat-square)](https://badge.fury.io/js/%40adobe%2Freact-native-aepedgeidentity)  |
+|  [EdgeConsent](https://aep-sdks.gitbook.io/docs/foundation-extensions/consent-for-edge-network)    |   [![npm version](https://img.shields.io/npm/v/@adobe/react-native-aepedgeconsent.svg?color=green&label=%40adobe%2Freact-native-aepedgeconsent&logo=npm&style=flat-square)](https://badge.fury.io/js/%40adobe%2Freact-native-aepedgeconsent)  |
+|  [Messaging]()  |   [![npm version](https://img.shields.io/npm/v/@adobe/react-native-aepmessaging.svg?color=green&label=%40adobe%2Freact-native-aepmessaging&logo=npm&style=flat-square)](https://badge.fury.io/js/%40adobe%2Freact-native-aepmessaging)   |
```

Comment:
Do you want to leave Messaging link blank?

Reply:
Yes, I want to keep it blank for now. The IAM PR is note merged https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/pull/714/files.  So I think we can let the messaging team decide which link should be put here before GA since